File: 1791-find-center-of-star-graph/1791-find-center-of-star-graph.java
class Solution {
    public int findCenter(int[][] edges) {
        Map<Integer, Integer> map = new HashMap<>();
        for (int i = 0; i < edges.length; i++) {
            if (map.containsKey(edges[i][0])) {
                map.put(edges[i][0], map.get(edges[i][0])+1);
            } else {
                map.put(edges[i][0], 1);
            }
            if (map.containsKey(edges[i][1])) {
                map.put(edges[i][1], map.get(edges[i][1])+1);
            } else {
                map.put(edges[i][1], 1);
            }
            if (map.get(edges[i][0]) >= 2) return edges[i][0];
            if (map.get(edges[i][1]) >= 2) return edges[i][1];
        }
        return 0;
    }
}

File: 1791-find-center-of-star-graph/1791-find-center-of-star-graph.py
class Solution(object):
    def findCenter(self, edges):
        """
        :type edges: List[List[int]]
        :rtype: int
        """
        edgecount = {}
        for edge in edges:
            if edge[0] not in edgecount.keys():
                edgecount[edge[0]] = 1
            else:
                edgecount[edge[0]] = edgecount[edge[0]] + 1
            if edge[1] not in edgecount.keys():
                edgecount[edge[1]] = 1
            else:
                edgecount[edge[1]] = edgecount[edge[1]] + 1
            if edgecount[edge[0]] >= 2:
                return edge[0]
            if edgecount[edge[1]] >= 2:
                return edge[1]
        return 0
        